### DISCUSSION TITLE
Limiter l'apparition de l'option d'appel des membres de la guilde lors d'une rencontre avec un animal sauvage aux personnes appartenant à une guilde

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#2123] [Nouvelles idées]
Ça me paraît bizarre d'avoir l'option :stadium: qui apparaît alors qu'un joueur peut ne faire partie d'aucune guilde lors d'une rencontre avec un animal sauvage (en gros, c'est comme un bon troll qui te dit "Tu veux de la rage? Bah t'en auras pas lul").

Un screen pour décrire ça en image: (j'ai pas pris l'option car je savais que ça ne servirait à rien au vu de mon côté solitaire)
![Screenshot_20230808-133225~2.png](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/94372084/16efae29-4280-4b56-9fb0-94922bd11a52)

Ça ne serait pas mieux de vérifier si le joueur appartient à une guilde avant de proposer l'option :stadium:, et la retirer s'il est solitaire?

**Comment by BastLast** [comment]
C'est fait exprès pr encourager les gens a y aller ensemble

**Reply by leuhvraiminoryo** [reply]
ouais enfin là pagos parle du cas où la personne n'est carrément pas dans une guilde, donc rien que la formulation "de votre guilde" est assez problématique. :eyes: 

(j'veux dire que t'façon, sans guilde, les gens ne peuvent pas "y aller ensemble")
